schema-loader/src/main/java/com/scalar/db/schemaloader/SchemaLoaderError.java

```diff
@@ -0,0 +1,141 @@
+      "",
+      ""),
+  ALTERING_CLUSTERING_ORDER_NOT_SUPPORTED(
+      Category.USER_ERROR,
+      "0003",
+      "The clustering ordering of the table %s.%s were modified, but altering clustering ordering is not supported",
+      "",
+      ""),
+  DELETING_COLUMN_NOT_SUPPORTED(
+      Category.USER_ERROR,
+      "0004",
+      "The column %s of the table %s.%s has been deleted. Column deletion is not supported when altering a table",
+      "",
+      ""),
+  ALTERING_COLUMN_DATA_TYPE_NOT_SUPPORTED(
+      Category.USER_ERROR,
+      "0005",
+      "The data type of the column %s of the table %s.%s was modified, but altering data types is not supported",
+      "",
+      ""),
+  SPECIFYING_SCHEMA_FILE_REQUIRED_WHEN_USING_REPAIR_ALL(
+      Category.USER_ERROR,
+      "0006",
+      "Specifying the '--schema-file' option is required when using the '--repair-all' option",
+      "",
+      ""),
+  SPECIFYING_SCHEMA_FILE_REQUIRED_WHEN_USING_ALTER(
+      Category.USER_ERROR,
+      "0007",
+      "Specifying the '--schema-file' option is required when using the '--alter' option",
+      "",
+      ""),
+  SPECIFYING_SCHEMA_FILE_REQUIRED_WHEN_USING_IMPORT(
+      Category.USER_ERROR,
+      "0008",
+      "Specifying the '--schema-file' option is required when using the '--import' option",
+      "",
+      ""),
+  SPECIFYING_COORDINATOR_WITH_IMPORT_NOT_ALLOWED(
+      Category.USER_ERROR,
+      "0009",
+      "Specifying the '--coordinator' option with the '--import' option is not allowed."
+          + " Create Coordinator tables separately",
+      "",
+      ""),
+  READING_CONFIG_FILE_FAILED(
+      Category.USER_ERROR, "0010", "Reading the configuration file failed. File: %s", "", ""),
+  READING_SCHEMA_FILE_FAILED(
+      Category.USER_ERROR, "0011", "Reading the schema file failed. File: %s", "", ""),
+  PARSING_SCHEMA_JSON_FAILED(
+      Category.USER_ERROR, "0012", "Parsing the schema JSON failed. Details: %s", "", ""),
+  PARSE_ERROR_TABLE_NAME_MUST_CONTAIN_NAMESPACE_AND_TABLE(
+      Category.USER_ERROR,
+      "0013",
+      "The table name must contain the namespace and the table. Table: %s",
+      "",
+      ""),
+  PARSE_ERROR_PARTITION_KEY_MUST_BE_SPECIFIED(
+      Category.USER_ERROR, "0014", "The partition key must be specified. Table: %s", "", ""),
+  PARSE_ERROR_INVALID_CLUSTERING_KEY_FORMAT(
+      Category.USER_ERROR,
+      "0015",
+      "Invalid clustering-key format. The clustering key must be in the format of 'column_name' or 'column_name ASC/DESC'."
+          + " Table: %s; Clustering key: %s",
+      "",
+      ""),
+  PARSE_ERROR_COLUMNS_NOT_SPECIFIED(
+      Category.USER_ERROR, "0016", "Columns must be specified. Table: %s", "", ""),
+  PARSE_ERROR_INVALID_COLUMN_TYPE(
+      Category.USER_ERROR, "0017", "Invalid column type. Table: %s; Column: %s; Type: %s", "", ""),
+  ;
+
+  private static final String COMPONENT_NAME = "DB-SCHEMA-LOADER";
```

Comment:
@feeblefakie The prefix of the error code for schema-loader is `DB-SCHEMA-LOADER`. Is that okay?